windows/hcsshimopts/hcsshim.proto

```diff
@@ -4,9 +4,23 @@ package containerd.windows.hcsshim;
 
 import "gogoproto/gogo.proto";
 import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
 
 option go_package = "github.com/containerd/containerd/windows/hcsshimopts;hcsshimopts";
```

Comment:
Looking at this, I think `hcsshimtypes` would be fine for this package name.

Reply:
no objections.